lib/common/pyDriveUtil.py

```diff
@@ -12,8 +13,12 @@ class PyDriveUtil(object):
     def __init__(self, settings=None):
         if settings is None:
             settings = {"settings_file": None, "local_cred_file": "mycreds.txt"}
-        self.gauth = self.get_gauth(settings)
-        self.drive = GoogleDrive(self.gauth)
+        if os.path.exists(settings['local_cred_file']):
+            self.gauth = self.get_gauth(settings)
+            self.drive = GoogleDrive(self.gauth)
+        else:
+            raise Exception(
+                "Your current working dir didn't inculde the client certificate file [%s], please make sure the firefox/chrome profile creation is turned off, online mode is disabled and not running the test cases of GSuite and FB!")
```

Comment:
Your current working dir `doesn't` `include` the client certificate file [%s]`.` Please make sure firefox/chrome profile creation is turned off, online mode is disabled`,` and `GSuite and FB cases are not test targets!`